### PR TITLE
zh-CN: update glossary/ajax

### PR DESCRIPTION
**Pull request by PassionPenguin**
### Description

catch up with the newest content

### CODE REVIEW

File: files/zh-cn/glossary/ajax/index.md [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
 {{GlossarySidebar}}
 
-**AJAX**（Asynchronous {{glossary("JavaScript")}} And {{glossary("XML")}} ）是一种使用 {{glossary("XMLHttpRequest")}} 技术构建更复杂，动态的网页的编程实践。
+**AJAX**（Asynchronous {{glossary("JavaScript")}} And {{glossary("XML")}} ）是一种在网页 App 中通过异步地发送 HTTP 请求向服务器获取内容，并使用这些新内容更新页面中相关的部份，而无需重新加载整个页面的网页开发技术。这可以让网页更具有响应性，因为只有那些需要更新的部分向服务器请求了。
 
-AJAX 允许只更新一个 {{glossary("HTML")}} 页面的部分 {{glossary("DOM")}}，而无须重新加载整个页面。AJAX 还允许异步工作，这意味着当网页的一部分正试图重新加载时，你的代码可以继续运行（相比之下，同步会阻止代码继续运行，直到这部分的网页完成重新加载）。
+AJAX 可以被用来创建 {{Glossary("SPA", "单页应用")}}，那些使用 AJAX 来更新需要的内容的、由单一文档构成的一整个网页 App。
 
-通过交互式网站和现代 Web 标准，AJAX 正在逐渐被 JavaScript 框架中的函数和官方的 {{domxref("Fetch API")}} 标准取代。
+一开始的时候 AJAX 通过使用 {{domxref("XMLHttpRequest")}} 接口实现，但是 {{domxref("fetch()")}} API 更适合用于开发现代网页 App：更出色、更灵活、更好地与一些网页 App 的基础技术相结合（如 [service workers](/zh-CN/docs/Web/API/Service_Worker_API)）。现代网页框架也为 AJAX 提供了抽象化概念。
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/ajax/index.md:14]:
```suggestion
一开始的时候 AJAX 通过使用 {{domxref("XMLHttpRequest")}} 接口实现，但是 {{domxref("fetch()")}} API 更适合用于开发现代网页 App：更出色、更灵活、更好地与一些网页 App 的基础技术相结合（如 [Service Workers](/zh-CN/docs/Web/API/Service_Worker_API)）。现代网页框架也为 AJAX 提供了抽象化概念。
```

**Review by PassionPenguin — COMMENTED** [review]
lint

File: files/zh-cn/glossary/ajax/index.md [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
 title: AJAX
 slug: Glossary/AJAX
+l10n:
+  sourceCommit: 5d6c210c89552b590f035e78b5c4f48cb48b32b6
 ---
 
 {{GlossarySidebar}}
 
-**AJAX**（Asynchronous {{glossary("JavaScript")}} And {{glossary("XML")}} ）是一种使用 {{glossary("XMLHttpRequest")}} 技术构建更复杂，动态的网页的编程实践。
+**AJAX**（Asynchronous {{glossary("JavaScript")}} And {{glossary("XML")}} ）是一种在网页 App 中通过异步地发送 HTTP 请求向服务器获取内容，并使用这些新内容更新页面中相关的部份，而无需重新加载整个页面的网页开发技术。这可以让网页更具有响应性，因为只有那些需要更新的部分向服务器请求了。
```
Comment (yin1999) [inline files/zh-cn/glossary/ajax/index.md:10]:
```suggestion
**AJAX**（Asynchronous {{glossary("JavaScript")}} And {{glossary("XML")}} ）是一种在 Web 应用中通过异步发送 HTTP 请求向服务器获取内容，并使用这些新内容更新页面中相关的部分，而无需重新加载整个页面的 Web 开发技术。这可以让网页更具有响应性，因为只请求了需要更新的部分。
```

File: files/zh-cn/glossary/ajax/index.md [hunk 3]
```diff
@@ -1,15 +1,19 @@
...
 
-AJAX 允许只更新一个 {{glossary("HTML")}} 页面的部分 {{glossary("DOM")}}，而无须重新加载整个页面。AJAX 还允许异步工作，这意味着当网页的一部分正试图重新加载时，你的代码可以继续运行（相比之下，同步会阻止代码继续运行，直到这部分的网页完成重新加载）。
+AJAX 可以被用来创建 {{Glossary("SPA", "单页应用")}}，那些使用 AJAX 来更新需要的内容的、由单一文档构成的一整个网页 App。
```
Comment (yin1999) [inline files/zh-cn/glossary/ajax/index.md:12]:
```suggestion
AJAX 可以用于创建{{Glossary("SPA", "单页应用")}}——那些使用 AJAX 来更新需要的内容的、由单一文档构成的一整个 Web 应用。
```

File: files/zh-cn/glossary/ajax/index.md [hunk 4]
```diff
@@ -1,15 +1,19 @@
...
 
-通过交互式网站和现代 Web 标准，AJAX 正在逐渐被 JavaScript 框架中的函数和官方的 {{domxref("Fetch API")}} 标准取代。
+一开始的时候 AJAX 通过使用 {{domxref("XMLHttpRequest")}} 接口实现，但是 {{domxref("fetch()")}} API 更适合用于开发现代网页 App：更出色、更灵活、更好地与一些网页 App 的基础技术相结合（如 [Service Workers](/zh-CN/docs/Web/API/Service_Worker_API)）。现代网页框架也为 AJAX 提供了抽象化概念。
```
Comment (yin1999) [inline files/zh-cn/glossary/ajax/index.md:14]:
```suggestion
一开始的时候 AJAX 通过使用 {{domxref("XMLHttpRequest")}} 接口实现，但是 {{domxref("fetch()")}} API 更适合用于开发现代 Web 应用：更出色、更灵活、更好地与一些 Web 应用的基础技术相结合（如 [Service Worker](/zh-CN/docs/Web/API/Service_Worker_API)）。现代 Web 框架也为 AJAX 提供了抽象。
```

File: files/zh-cn/glossary/ajax/index.md [hunk 5]
```diff
@@ -1,15 +1,19 @@
...
+
+AJAX 在现代网页开发中太过常见了，以至于 AJAX 这个术语很少被使用到。
```
Comment (yin1999) [inline files/zh-cn/glossary/ajax/index.md:16]:
```suggestion
这项技术在现代 Web 开发中太过常见了，以至于 AJAX 这个术语很少被使用到。
```

**Review by yin1999 — COMMENTED** [review]
We would not translation the word "web"

**Review by yin1999 — APPROVED** [review]
Thank you @PassionPenguin

**Pull request merged**